oneview_redfish_toolkit/tests/test_util.py

```diff
@@ -295,14 +306,16 @@ def test_submit_event_with_subscriber(
 
         self.assertTrue(start_mock.call_count == 2)
 
+    @mock.patch.object(util, 'check_oneview_availability')
     @mock.patch.object(util, 'OneViewClient')
```

Comment:
Check if we still need this OneViewClient mock.

Reply:
We don't need it, once load_config() does not instantiate it anymore. Removing it.